src/io/flutter/logging/FlutterLogView.java

```diff
@@ -39,8 +37,32 @@
 import java.awt.datatransfer.StringSelection;
 import java.awt.event.MouseEvent;
 
+import static com.intellij.ui.SimpleTextAttributes.STYLE_PLAIN;
+
 public class FlutterLogView extends JPanel implements ConsoleView, DataProvider, FlutterLog.Listener {
 
+  private class EntryModel implements FlutterLogTree.EntryModel {
+    boolean color;
+
+    @Override
+    public SimpleTextAttributes style(@Nullable FlutterLogEntry entry, int attributes) {
+      if (color && entry != null) {
+        final FlutterLog.Level level = FlutterLog.Level.forValue(entry.getLevel());
+        if (level != null) {
```

Comment:
earlier return too.

Reply:
Cool.  Thanks.  See above for my rationale.